apps/src/templates/sectionAssessments/CorrectAnswerIcon.story.jsx

```diff
@@ -6,9 +6,15 @@ export default storybook => {
     .storiesOf('SectionAssessments/CorrectAnswerIcon', module)
     .addStoryTable([
       {
-        name:'CorrectAnswerIcon',
+        name: 'CorrectAnswerIcon',
+        description: `
+            A green check mark is visible if students answer a question
+            correctly.
+        `,
         story: () => (
-          <CorrectAnswerIcon/>
+          <CorrectAnswerIcon
+            percentValue="40%"
+          />
         )
```

Comment:
Please add another story with an example of how to use the `isCorrectAnswer` prop that you defined.

Reply:
Added story.